docs/user/run-node/07-multisig.md

```diff
@@ -0,0 +1,89 @@
+
+This is done by signing the transaction multiple times, once with each private key. The order of the signatures matters and must match the order of the public keys in the multisig key.
+
+Once you have a transaction with the necessary signatures, it can be broadcasted to the network. The network will verify that the transaction has the necessary signatures from the accounts in the multisig key before it is executed.
+
+## Step by step guide to multisig transactions
+
+Check the keys you have in `--keyring-backend test`.
+
+```shell
+simd keys list --keyring-backend test
+````
+
+If you don't already have accounts listed create the accounts using the below.
+```shell
+simd keys add alice --keyring-backend test
+simd keys add bob --keyring-backend test
+simd keys add recipient --keyring-backend test
+```
+
+Create the multisig account between bob and alice.
+```shell 
+simd keys add alice-bob-multisig --multisig alice,bob --multisig-threshold 2 --keyring-backend test
+```
+
+Before initiating any transaction, verify the balance of each account and note down the amount. This step is crucial to confirm that the transaction can be processed successfully.
+```shell
+simd query bank balances $(simd keys show my_validator --keyring-backend test -a)
+simd query bank balances $(simd keys show alice-bob-multisig --keyring-backend test -a)
+```
+
+Ensure that the alice-bob-multisig account has funds. In our case, the genesis account, my_validator, holds our funds. Therefore, we will transfer funds from the `my_validator` account to the `alice-bob-multisig` account.
+
+The below command adds funds to the multisig.
+```shell
+simd tx bank send $(simd keys show my_validator --keyring-backend test -a) $(simd keys show alice-bob-multisig --keyring-backend test -a) "10000stake" --keyring-backend test
+```
+
+Check both accounts again to see if the funds have transferred.
+```shell
+simd query bank balances $(simd keys show alice-bob-multisig --keyring-backend test -a)
+```
+
+Initiate the transaction. This command will create a transaction from the multisignature account `alice-bob-multisig` to send 1000stake to the recipient account. The transaction will be generated but not broadcasted yet.
+```shell
+simd tx bank send $(simd keys show alice-bob-multisig --keyring-backend test -a) $(simd keys show recipient --keyring-backend test -a) 1000stake --generate-only --chain-id my-test-chain --keyring-backend test > tx.json
+```
+
+Alice signs the transaction using her key and refers to the multisig address. Execute the command below to accomplish this:
+```shell
+simd tx sign --from $(simd keys show alice --keyring-backend test -a) --multisig=cosmos1re6mg24kvzjzmwmly3dqrqzdkruxwvctw8wwds tx.json --sign-mode amino-json --chain-id my-test-chain --keyring-backend test > tx-signed-alice.json
```

Comment:
Do you need to specify the sign mode for this?

Reply:
fine without, good eye